content/release-notes/2023-03-10.md

```diff
@@ -0,0 +1,14 @@
+---
+label: 'Console'
```

Comment:
Actually, the Release wasn't rolled out due to a bug in Control Plan. 
So I would put everything into the `2023-03-14` notes except "Reduced the number of queries performed". It was reverted and has not merged with fix yet.

Reply:
Thank you. Moved all notes in the 03-14 notes. Dropped the reverted item.